src/m_misc.c

```diff
@@ -2424,6 +2424,14 @@ default_t defaults[] = {
     "1 to enable uncapped rendering frame rate"
   },
 
+  // framerate limit
+  {
+    "fpslimit",
+    (config_t *) &fpslimit, NULL,
+    {0}, {35, 500}, number, ss_gen, wad_no,
```

Comment:
I'm afraid this won't work. The default value has to be in the min-max range.

Reply:
Drat! How about I switch `uncapped` to a multivalue where 0 = off, 1 = uncapped with no limit, 2 = uncapped with limit?

Reply:
In the latter case, we could even uncap the upper limit. We should also mention in the description that this requires uncapped framerate enabled.

Reply:
Sorry, I should have elaborated more. I definitely intend to expose this setting in the menu. There would be a separate line item to set the value for `fpslimit` and the `uncapped` menu item would grow from two options (OFF and ON) to three (OFF, ON no cap, ON with cap). If `uncapped` is set to OFF or ON no cap then I would gray out the `fpslimit` menu item.

The advantage to this is that we can cleanly specify the min-max range for `fpslimit` (no shenanigans with using a value of 0 to disable) and it does not disturb existing configs.

Reply:
Yes, this sounds like the best solution. The menu will need some rearrangement then (yay, it's high time, we didn't have this for literally *weeks* 😉).

Reply:
I haven't dug into the Woof! menu code yet, but maybe we could do something without adding a new menu item. Could we perhaps have a press of the enter key bring up a prompt to allow the user to key in the frame limit value?

Reply:
Yes, of course. The Woof! menu has already inherited numerical input from Boom/MBF. For example, you may set a numerical value for game speed in percent.